### PR TITLE
Update install-kubeadm.md

### PR DESCRIPTION
**Pull request by Pensu**
Add note to kubeadm install instruction to help install in other archs i.e. aarch64, ppc64le etc.

### CODE REVIEW

File: docs/setup/independent/install-kubeadm.md [hunk 1]
```diff
@@ -215,6 +215,7 @@ systemctl enable kubelet && systemctl start kubelet
     EOF
     sysctl --system
     ```
+  - Replace x86_64 in baseurl with your system's arch to install kubeadm in other archs i.e aarch64, ppc64le etc.
```
Comment (xiangpengzhao) [inline docs/setup/independent/install-kubeadm.md]: SGTM. Maybe we want to list all of the supported arch here?

**Comment by Pensu** [comment]
@xiangpengzhao I have updated the baseurl. Now, it will take the parameter $baseurl which will automatically use the platform's arch. We don't need a note now to put anything else specifically. PTAL.

**Comment by xiangpengzhao** [comment]
@Pensu where is the `$basearch` defined or where can its value be gotten ? I think I'm missing something here :)

/cc @kubernetes/sig-cluster-lifecycle-pr-reviews

**Comment by Pensu** [comment]
@xiangpengzhao The value of $basearch variable is defined by yum in the runtime by calling uname function. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/sec-using_yum_variables

**Comment by kad** [comment]
looks good, but please cleanup trailing spaces.

**Comment by Pensu** [comment]
@kad Sorry about that! Fixed it, please check.

**Comment by steveperry-53** [comment]
/approve (for writing)
Still need `/lgtm` for technical accuracy.

**Comment by zacharysarah** [comment]
@xiangpengzhao 👋 Any more review or changes needed?

**Comment by xiangpengzhao** [comment]
/lgtm
as per the referenced link provided by @Pensu :) :+1:

**Pull request merged**